src/api/local/deployTools.ts

```diff
@@ -72,9 +72,24 @@ export namespace DeployTools {
 
           methods.push({ method: "all" as DeploymentMethod, label: `All`, description: `Every file in the local workspace` });
 
-          method = (await vscode.window.showQuickPick(methods,
-            { placeHolder: `Select deployment method to ${remotePath}` }
-          ))?.method;
+          let defaultDeploymentMethod = instance.getConfig()?.defaultDeploymentMethod as DeploymentMethod
```

Comment:
You can (and should) use `const` here as `defaultDeploymentMethod` isn't changed after being set.
```suggestion
          const defaultDeploymentMethod = instance.getConfig()?.defaultDeploymentMethod as DeploymentMethod
```